blocks/bricks/sequence_generators.py

```diff
@@ -203,6 +208,13 @@ def cost_matrix(self, application_call, outputs, mask=None, **kwargs):
             mask=mask, return_initial_states=True, as_dict=True,
             **dict_union(inputs, states, contexts))
 
+        # Run the language model
+        if self.language_model:
+            lm_arguments = self.language_model.apply(
```

Comment:
Just a reminder,  in the end we should be able to accept language model states from the user to prevent their re-computation. But for now it has low priority.